<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BugReport</class>
 <widget class="QDialog" name="BugReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bug Report</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/services/images/services/invalid.png</normaloff>:/services/images/services/invalid.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string>QFrame{background: white;} </string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="imgError">
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources/resources.qrc">:/error/images/errorVideo.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lblErrorInfo">
        <property name="text">
         <string>&lt;b&gt;Ops! An error occurred while trying to download the video.&lt;/b&gt;
&lt;p&gt;You can help to fix this problem sending us an &quot;&lt;b&gt;Bug report&lt;/b&gt;&quot;. This Report will be used to analyze the error and if is necessary fix it and improve the next version of xVST.&lt;/p&gt;
&lt;p&gt;Your help is allways welcomed!&lt;/p&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;p&gt;The &quot;&lt;b&gt;Bug report&lt;/b&gt;&quot; will be sent to &lt;a href=&quot;http://www.xvideothief.com/tracker/&quot;&gt;xVST  tracker&lt;/a&gt;.&lt;/p&gt;
&lt;p&gt;Your name and email will be included into the &quot;&lt;b&gt;Bug comment&lt;/b&gt;&quot;, however is totally optional and unecessary.&lt;/p&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="gpbErrorInfo">
     <property name="title">
      <string>Error information</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0" colspan="3">
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="lblVideoURL">
          <property name="text">
           <string>Video URL: &lt;b&gt;%1&lt;/b&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblxVSTVersion">
          <property name="text">
           <string>xVST version: &lt;b&gt;%1&lt;/b&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="Line" name="line">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="lblName">
          <property name="text">
           <string>Your name: &lt;span style=&quot; font-style:italic; color:#a9a9a9;&quot;&gt;(optional)&lt;/span&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtName">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>21</height>
           </size>
          </property>
          <property name="whatsThis">
           <string>Your name will be added into the &quot;Bug report&quot; body. This field is totally optional</string>
          </property>
          <property name="accessibleName">
           <string>Your name</string>
          </property>
          <property name="accessibleDescription">
           <string>Your name will be added into the &quot;Bug report&quot; body. This field is totally optional</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="lblEmail">
          <property name="text">
           <string>Email: &lt;span style=&quot; font-style:italic; color:#a9a9a9;&quot;&gt;(optional)&lt;/span&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtEmail">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>21</height>
           </size>
          </property>
          <property name="whatsThis">
           <string>Your email will be added into the &quot;Bug report&quot; body. This field is totally optional</string>
          </property>
          <property name="accessibleName">
           <string>Your email</string>
          </property>
          <property name="accessibleDescription">
           <string>Your email will be added into the &quot;Bug report&quot; body. This field is totally optional</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="2">
       <layout class="QVBoxLayout">
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="spbViewInfo">
          <property name="minimumSize">
           <size>
            <width>23</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>23</width>
            <height>23</height>
           </size>
          </property>
          <property name="toolTip">
           <string>View which information will be sent</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources/resources.qrc">
            <normaloff>:/error/images/view.png</normaloff>:/error/images/view.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="3">
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="lblComment">
          <property name="text">
           <string>Your comments: &lt;span style=&quot; font-style:italic; color:#a9a9a9;&quot;&gt;(optional)&lt;/span&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="rchComments">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>28</height>
           </size>
          </property>
          <property name="whatsThis">
           <string>You can add your own comment which will be added into the &quot;Bug report&quot; body. This field is totally optional</string>
          </property>
          <property name="accessibleName">
           <string>Your comments</string>
          </property>
          <property name="accessibleDescription">
           <string>You can add your own comment which will be added into the &quot;Bug report&quot; body. This field is totally optional</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="Line" name="line_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="lblSending">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Sending your &quot;&lt;b&gt;Bug report&lt;/b&gt;&quot;... please wait...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="prbSending">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximum">
           <number>0</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="chbNoErrors">
       <property name="whatsThis">
        <string>If this option is checked, this dialog will not be displayed again (and no &quot;Bug report&quot; will be sent)</string>
       </property>
       <property name="accessibleName">
        <string>Do not display &quot;Bug Report&quot;</string>
       </property>
       <property name="accessibleDescription">
        <string>If this option is checked, this dialog will not be displayed again (and no &quot;Bug report&quot; will be sent)</string>
       </property>
       <property name="text">
        <string>Don't display &quot;Bug Report&quot; the next time</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>49</width>
         <height>25</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnSend">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="whatsThis">
        <string>Send the &quot;Bug report&quot; and close this dialog</string>
       </property>
       <property name="accessibleName">
        <string>Send button</string>
       </property>
       <property name="accessibleDescription">
        <string>Send the &quot;Bug report&quot; and close this dialog</string>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="whatsThis">
        <string>Close this dialog without sending the &quot;Bug report&quot;</string>
       </property>
       <property name="accessibleName">
        <string>Cancel button</string>
       </property>
       <property name="accessibleDescription">
        <string>Close this dialog without sending the &quot;Bug report&quot;</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtName</tabstop>
  <tabstop>edtEmail</tabstop>
  <tabstop>spbViewInfo</tabstop>
  <tabstop>rchComments</tabstop>
  <tabstop>chbNoErrors</tabstop>
  <tabstop>btnSend</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
